### ISSUE TITLE
Ghost component with styling?

### ISSUE THREAD

**Issue by theodol59** [formkit/drag-and-drop#62]
Are there any plans to have custom ghost components? Default native ghost component is barely visible.. It would be cool to have more visible ghost (active draggable) component with custom styling abilities

**Comment by sashamilenkovic** [comment]
@theodol59 Definitely! One of my main _todos_. We should have a workaround for that.

**Comment by sashamilenkovic** [comment]
As of v0.2.0, as a workaround, you can now set `nativeDrag` to false to use synthetic dragging instead. I'm going to close this in favor of #54
